api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -568,6 +574,9 @@ public List<ReportingWorkspace> getWorkspaceBatch(long limit, long offset) {
             + "  JOIN cdr_version c ON w.cdr_version_id = c.cdr_version_id\n"
             + "  JOIN access_tier a ON c.access_tier = a.access_tier_id\n"
             + "  LEFT OUTER JOIN featured_workspace fw ON w.workspace_id = fw.workspace_id\n"
+            + "  LEFT OUTER JOIN user_initial_credits_expiration uice ON uice.user_id = creator_id\n"
```

Comment:
This is an outer join, because currently, if a user does not have an expiration record, they are considered unexpired.